lib/network_config.ml

```diff
@@ -33,6 +33,7 @@ let read_management_conf () =
 		let args = List.map (fun s -> match (String.split '=' s) with k :: [v] -> k, String.strip ((=) '\'') v | _ -> "", "") args in
 		debug "Firstboot file management.conf has: %s" (String.concat "; " (List.map (fun (k, v) -> k ^ "=" ^ v) args));
 		let device = List.assoc "LABEL" args in
+		let vlan = if List.mem_assoc "VLAN" args then List.assoc "VLAN" args else "" in
 		Inventory.reread_inventory ();
 		let bridge_name = Inventory.lookup Inventory._management_interface in
```

Comment:
Does the host installer still write this key? I thought it wouldn't, and then this will fail.